local-gems/spectrum-json/lib/spectrum/response/holdings.rb

```diff
@@ -23,16 +22,16 @@ def renderable
       def process_response
         data = []
         sorter = Hash.new { |hash, key| hash[key] = key }.tap do |hash|
-          hash[nil] = 'AAAC'
-          hash['Online Resources'] = 'AAAA'
-          hash['HathiTrust Digital Library'] = 'AAAB'
-          hash['- Offsite Shelving -'] = 'zzzz'
+          hash[nil] = "AAAC"
+          hash["Online Resources"] = "AAAA"
+          hash["HathiTrust Digital Library"] = "AAAB"
+          hash["- Offsite Shelving -"] = "zzzz"
         end
         @holdings.each do |holding|
           holding_presenter = @holding_factory.call(holding)
-          if holding_presenter.class.to_s.match?(/LinkedHolding/) || holding.items.count > 0
-            data << holding_presenter.to_h          
-          end
+          # if holding_presenter.class.to_s.match?(/LinkedHolding/) || holding_presenter.type == "empty" || holding.items.count > 0
+          data << holding_presenter.to_h
+          # end
         end
         data = data.reject do |item|
```

Comment:
If we're done doing this logic here, I would advocate a `@holdings.map` call instead of `@holdings.each`.

```ruby
data = @holdings.map {|holding| @holding_factory.call(holding).to_h}.reject do |item|
```

That said, it's fine the way it is, though.